src/webots/nodes/WbRobot.hpp

```diff
@@ -74,6 +74,7 @@ class WbRobot : public WbSolid {
   bool isWaitingForWindow() const { return mWaitingForWindow; }
   void setWaitingForWindow(bool waiting);
   void addNewlyInsertedDevice(WbNode *node);
+  void updatingMissingResources() const override;
```

Comment:
We usually don't use the progressive tense form (-ing) for methods, I would suggest to rename this method with a present tense form, e.g. without -ing:
```suggestion
  void fixMissingResources() const override;
```